src/test/simple_test.tcc

```diff
@@ -0,0 +1,58 @@
+
+namespace libzeth
+{
+namespace test
+{
+
+template <typename ppT>
+void simple_circuit(libsnark::protoboard<libff::Fr<ppT>> &pb)
+{
+    using namespace libsnark;
+    using FieldT = libff::Fr<ppT>;
```

Comment:
Let's use `FieldT` as a template, instead of `ppT` (you define it in the `simple_test.cpp` file and use it with the protoboard)

Reply:
Changed the template parameter